vendor/sheet2linkml/sheet2linkml/cli.py

```diff
@@ -87,9 +99,8 @@ def main(filter_entity, logging_config, write_mappings, include_terminologies):
         mappings = list()
 
         for entity in selected_entities:
-            filename = f"output/{entity.get_filename()}.yaml"
-            logging.info(f"Writing entity {entity.name} to {filename}")
-            yaml_dumper.dump(entity.as_linkml(crdch_root), filename)
+            logging.info(f"Writing entity {entity.name} to {output_filename}")
```

Comment:
Ideally you'd want to create a logger for this module and use the logger with its various logging levels, like `info` and such. Not sure if you want to consider using the [click_log](https://github.com/click-contrib/click-log) library too?

```python
logger = logging.getLogger(__name__)
click_log.basic_config(logger)

# code for module

logger.info(f"{}")
```

If `logging` is used in other modules too it might grow into a large PR which we should ideally avoid. Happy to create a new issue from this and tackle it if you think it's worthwhile.

Reply:
I'm not very good with formal logging and defer to @sujaypatil96 on this.

Reply:
Good idea! I've filed this as https://github.com/cancerDHC/ccdhmodel/issues/106.